pom.xml

```diff
@@ -25,7 +25,9 @@
         </developer>
     </developers>
     <properties>
-        <workflow.version>1.3</workflow.version>
+        <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <workflow.version>1.4.3-beta-1</workflow.version>
```

Comment:
Maybe we should start adding a CHANGELOG.txt on each Pipeline module. It's going to be difficult to remember why 1.4.3 exists.